Design/Rule0023AlwaysSpecifyFieldgroups.cs

```diff
@@ -20,6 +21,10 @@ private void CheckFieldgroups(SymbolAnalysisContext ctx)
             try
             {
                 ITableTypeSymbol table = (ITableTypeSymbol)ctx.Symbol;
+                if (table.Length > 0)
```

Comment:
Under which condition is table length not more than 0? What does this condition catch?

Reply:
Oops, this is a typo and should be table.Fields.Length. Fixed with the change to the PrimaryKey (table.PrimaryKey.Fields.Length != 1)